src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../services/user';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  setTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(todo.userId)
+      .then(data => {
+        setUser(data);
+      })
+      .catch(() => {
+        setUser(null);
```

Comment:
The `catch` block is empty and only sets the user to `null`. Consider logging the error or providing an error message to help with debugging.